agent/ui_endpoint.go

```diff
@@ -395,7 +417,7 @@ func summarizeServices(dump structs.ServiceDump, cfg *config.RuntimeConfig, dc s
 		hasProxy = make(map[structs.ServiceName]bool)
```

Comment:
`hasProxy` and `summary` should be keyed on `structs.PeeredServiceName`, otherwise the peering information would be lost